regex/icu/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I would really love to see us:

1) Move this module out of go-mysql-server into its own library.
2) Have some documentation around what version of emsdk this is coming from.
3) Have some documentation around where `icudt68l.dat` came from.

Ideal world would have all the following in a GitHub action we could run:

1) Download emsdk.
2) Activate it.
3) Compile test.cpp with the right exported functions, compiler flags, etc.
4) Drop the icu.wasm and the icudt68l.dat into the correct place in the repository.
5) If there were changes (will there always be changes? reproducible builds ftw?!), push the commit main/a PR.

But just having it a bit better documented is a good start...at least versions for the emsdk installation, for example.